go.mod

```diff
@@ -5,9 +5,9 @@ go 1.21
 require (
 	github.com/Azure/azure-kusto-go v0.15.0
 	github.com/Azure/azure-sdk-for-go v68.0.0+incompatible
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.9.1
-	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.4.0
-	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v5 v5.4.0
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.9.2
```

Comment:
do we need to bump for master and other branches?

Reply:
only master branch is needed. https://github.com/kubernetes-sigs/cloud-provider-azure/pull/5440